deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -0,0 +1,45 @@
+{
+    "jobDefinitionName": "production-pfb-analysis-run-job",
+    "type": "container",
+    "parameters": {},
+    "containerProperties": {
+        "image": "{image}",
+        "vcpus": 8,
```

Comment:
We could very safely drop this to 7 vcpus so that one CPU and enough mem remains available for instances to concurrently process tiling jobs as well. Thoughts?